Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -18,6 +18,26 @@ using namespace Gen;
 
 void EmuCodeBlock::MemoryExceptionCheck()
 {
+  // TODO: We really should untangle the trampolines, exception handlers and
+  // memory checks.
```

Comment:
Yes... it should be easier to refactor with this patch: I think this patch eliminates all the callers outside of JitUtil.cpp?